util/a2e/atoe_utils.c

```diff
@@ -291,7 +295,7 @@ fnumber(InstanceData *this, long value, int format_type, int left_justify,
  *               left_justify  Left justify number flag
  *               min_width     Minimum number of charaters value will
  *                             occupy
- *               precision
+ *               precision     Fix the width with leading zeros
```

Comment:
Again, this parameter is about the width with padding. Whether that padding is spaces or zeros is determined by `zero_pad`.